cypress/e2e/pages/storyPage/index.cy.js

```diff
@@ -222,17 +221,6 @@ const liteTestSuites = canonicalTestSuites
   });
 
 describe('storyPage', () => {
```

Comment:
As above, we can delete the `describe` now - it was only needed because we had a `beforeEach`